### PR TITLE
Fixin' yo stuff

### PR DESCRIPTION
**Pull request by nino**
ftfy

### CODE REVIEW

**Comment by nino** [comment]
important

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ An example of an extension can be found here: [Jarvis Extension](https://github.
 To get started, clone the repository and install the required dependencies:
 
 ```
-git clone https://github.com/vanguard12/Jarvis.git
+git clone https://github.com/Ewan-Roberts/Jarvis.git
```
Comment (Ewan-Roberts) [inline README.md:14]: Nay, this is my human name, vanguard12 is my level 12 skeleton archer.

Nice try elven warlock, I see your mind tricks. Advice for next time:

image

Comment (nino) [reply README.md:14]: I thought you were the elven warlock?

Comment (Ewan-Roberts) [reply README.md:14]: I specced too much into dexterity and not enough into... racial consistency? 

I am now re-rolling as a pretty pretty unicorn

PR APPROVED!

**Pull request merged**